Test/Test.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\TorchSharp\TorchSharp.csproj" />
+    <ProjectReference Include="..\SNT\SNT.csproj" />

```

Comment:
Maybe we should have our own test project for the SNT wrapper? That way, we keep the two layers completely separate.

Reply:
I agree. At this point we should address #34 and create a unique test folder (either in source or on its own).